Sources/Fullscreen/Explore/ExploreView.swift

```diff
@@ -187,7 +266,7 @@ extension ExploreView: UICollectionViewDelegate {
 extension ExploreView: TagCloudGridViewDelegate {
     public func tagCloudGridView(_ view: TagCloudGridView, didSelectItem item: TagCloudCellViewModel, at indexPath: IndexPath) {
         let indexPath = IndexPath(item: indexPath.item, section: view.tag)
-        let item = sections[indexPath.section].items[indexPath.item]
+        let item = exploreSections[indexPath.section].items[indexPath.item]
```

Comment:
Again, this only works if the list of sections are pre-sorted before being provided to this view. Is it possible to use the `sections` property instead?

Reply:
The TagCloudGridView will always be an exploreSection, but I can see how I'm introducing future confusion for recommerce developers by not being clearer.

Reply:
I've added a comment here as this was not super clear to me either - this was old code. From comments and rewrite of the other didSelectItem func this should now at least be easier to read.